packages/core/src/asset/ResourceManager.ts

```diff
@@ -5,6 +5,8 @@ import { Loader } from "./Loader";
 import { LoadItem } from "./LoadItem";
 import { RefObject } from "./RefObject";
 
+type EditorResourceItem = { virtualPath: string; path: string; type: string; id: string };
```

Comment:
Add comments with `@beta  just for internal editor, not recommended for developers`